packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -255,6 +255,13 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
    */
   readonly machineImage: ec2.IMachineImage;
 
+  /**
+   * Security group to launch the instances in.
+   *
+   * @default - A SecurityGroup will be created if none is specified.
+   */
+  readonly securityGroup?: ec2.ISecurityGroup;
```

Comment:
This options should have been placed into the `CommonAutoScalingGroupProps` instead of the `AutoScalingGroupProps` because there is still the issue with ECS cluster when adding a capacity. The `AddCapacityOptions` does not support the new `securityGroup` option.